lib/diagnostic.rb

```diff
@@ -91,13 +94,14 @@
 # In a Ruby string, write how you would remove the last two elements from this
 # array
 
-Diagnostic.remove_from_array = 'your answer here'
+Diagnostic.remove_from_array = 'use delete method to select "56" and "67"'
 
 # Taking the result from the question above, what would `arr` now equal:
 arr[arr.length + 2] = 99
 # Write your response as a ruby string
 
-Diagnostic.array_add_to = 'your answer here'
+Diagnostic.array_add_to = 'It would not add 99 because that index does not exist
+at that length'
 
```

Comment:
we were looking for what that code would result in.

`'[12, 34, 56, 67, nil, nil, 99]'`

Always try running code like this yourself to see the return.